static/app/views/insights/sessions/charts/newAndResolvedIssueChart.tsx

```diff
@@ -12,7 +14,12 @@ export default function NewAndResolvedIssueChart({type}: {type: 'issue' | 'feedb
 
   return (
     <InsightsBarChartWidget
-      title={type === 'issue' ? t('Issues') : t('User Feedback')}
+      title={
+        type === 'issue'
+          ? CHART_TITLES.NewAndResolvedIssueChart_issue
+          : t('User Feedback')
+      }
+      interactiveTitle={() => <ChartSelectionTitle />}
```

Comment:
😭 all that work putting the chart titles in a constant, but `t('User Feedback')` isn't in it! Honestly might make sense to export `NewAndResolvesIssueChart` and `NewAndResolvedUserFeedback` chart separately, and then there won't be that strange `'NewAndResolvedIssueChart_issue'` key